suggest/pathogens.txt

```diff
@@ -0,0 +1,296 @@
+55,Gonorrhea
+264,Guillain Barre
+56,Guinea Worm
+222,H. influenzae
+59,H. pylori
+169,Hand
+58,Hantavirus
+261,Helminths
+151,Hemorrhagic Fever
+60,Hepatitis
+243,Hepatitis A
+244,Hepatitis B
+245,Hepatitis C
+246,Hepatitis D
+247,Hepatitis E
+61,Herpes
+62,Herpes Zoster
+266,HFM-CoxsackieA
+265,HFM-Enterovirus71
+63,Histoplasmosis
+2,HIV/AIDS
+64,Hookworm
+94,Hospital-Related Infection
+65,HPV
+172,Human Bocavirus
+66,Hymenolepiasis
+67,Influenza
+224,Influenza H1N1
+268,Influenza H3N2
+69,Isosporiasis
+187,Japanese Encephalitis
+269,Kawasaki Disease
+272,Klebsiella pneumoniae
+71,Kuru
+301,Kyasanur Forest Disease
+240,La Crosse Encephalitis
+72,Lassa Fever
+73,Legionnaires'
+74,Leishmaniasis
+75,Leprosy
+76,Leptospirosis
+163,LGV
+98,Lice
+77,Listeriosis
+21,Liver Fluke
+291,Loiasis
+186,Lumpy Skin Disease
+78,Lyme Disease
+79,Machupo Virus
+80,Malaria
+81,Marburg Fever
+82,Measles
+83,Melioidosis
+84,Meningitis
+227,Meningitis - HiB
+229,Meningitis - Neisseria
+228,Meningitis - Strep/Pneumoccocal
+85,Meningococcemia
+285,MERS
+86,Metagonimiasis
+88,Monkeypox
+89,Mononucleosis
+87,MRSA
+90,Mucormycosis
+91,Mumps
+253,Murray Valley encephalitis
+166,Mycobacteria
+92,Myiasis
+157,Newcastle Disease
+214,Nipah/Hendra Virus
+93,Norovirus
+174,Not Yet Classified
+286,Novel Swine Enteric Coronavirus
+95,Onchocerciasis
+199,Other Animal Disease
+249,Other Human Disease
+200,Other Plant Disease
+96,Otitis media
+171,P. aeruginosa
+233,Paracoccidioidomycosis
+188,Paragonimus
+241,Paralytic Shellfish Poisoning
+97,Parotitis
+180,Parvovirus
+181,Pasteurellosis
+99,Pelvic Inflammatory Disease
+190,Peste des petits ruminants
+197,Pests
+101,Pinworm
+102,Plague
+305,PMSIS
+103,Pneumococcal Pneumonia
+104,Pneumonia
+176,Poisoning
+105,Polio
+185,Porcine Circovirus
+284,Powassan virus
+106,Prion
+276,Proteus mirabilis
+262,Protozoa
+217,Psittacosis
+108,Q Fever
+109,Rabies
+196,Rash
+238,Relapsing Fever
+212,Respiratory Illness
+110,Respiratory Syncytial Virus
+111,Rheumatic Fever
+112,Ricin
+225,Rickettsia
+175,Rift Valley Fever
+255,Ringworm
+114,River Blindness
+115,Rocky Mountain Spotted Fever
+116,Roseola
+117,Ross River Virus
+189,Rotavirus
+118,Rubella
+119,Salmonella
+121,SARS
+122,Scabies
+123,Scarlet Fever
+124,Schistosomiasis
+250,Schmallenberg
+194,Scrapie
+251,Scrub Typhus
+168,Sepsis
+274,Serratia marcescens
+125,Sexually Transmitted Infection
+182,Sheep/Goat Pox
+126,Shigellosis
+127,Shingles
+128,Smallpox-Related
+198,Soybean Rust
+234,Sporotrichosis
+236,St. Louis Encephalitis
+129,Staph
+130,Staphylococcal Entertoxin B
+277,Stenotrophomonas maltophilia
+179,Strangles
+131,Strep
+160,Strep Group B
+132,Syphilis
+133,Taeniasis
+134,Tetanus
+15,Thrush
+215,Tick-borne disease
+239,Tick-borne Encephalitis
+136,Toxic Shock Syndrome
+137,Toxocariasis
+138,Toxoplasmosis
+139,Trachoma
+140,Trichinellosis
+141,Trichinosis
+142,Trichuriasis
+143,Trypanosomiasis
+144,Tuberculosis
+145,Tularemia
+120,Typhoid
+113,Typhus
+162,Undiagnosed
```

Comment:
is that a pathogen?

Reply:
I've asked Stephen here https://docs.google.com/spreadsheets/d/1qOnGJQg1gtbmNxAW8c9hC1TvFpIQmSnORD-x-iXrpJ0/edit#gid=1945986228 and pinged as well. I'll include it for now since it's in the sheet, but easy to remove if Stephen says it should be.